modules/code-editor/src/views/full/index.tsx

```diff
@@ -81,6 +82,30 @@ export default class CodeEditor extends React.Component<Props, State> {
     this.setState({ isEditing: false, editedContent: undefined }, this.initialize)
   }
 
+  removeFile = async (file: EditableFile) => {
+    await this.props.bp.axios.post('/mod/code-editor/remove', {
+      ...file
+    })
+    await this.initialize()
+  }
```

Comment:
There should be a confirm before deleting anything. Also, you could just pass directly file instead of deconstructing it